lua/astrocommunity/editing-support/vim-move/init.lua

```diff
@@ -1,4 +1,4 @@
 return {
   "matze/vim-move",
-  event = "BufEnter",
+  event = "User AstroFile",
```

Comment:
```suggestion
  event = { "User AstroFile", "InsertEnter" },
```

This lets it work on new files as well